test/integration/narrow/routes/collaboration.test.js

```diff
@@ -1,6 +1,10 @@
+const { getCookieHeader, getCrumbCookie } = require('./test-helper')
 describe('Collaboration page', () => {
+  process.env.COOKIE_PASSWORD = '1234567890123456789012345678901234567890'
```

Comment:
This is not needed as it's set in `docker-compose.yaml`. Same for all the other tests below.